### ISSUE TITLE
Use syntax type to define if command is enabled.

### ISSUE THREAD

**Issue by skuroda** [tssajo/Minify#16]
I've recently come across a case where I have JSON file that doesn't contain one of the expected file extensions. As ST allows me to apply whatever syntax highlighting I want, it would be convenient if the functionality of your plugin supported it as well. 

Relevant API if you choose to implement this: `view.settings().get('syntax')`

**Comment by qkdreyer** [comment]
+1

**Comment by tssajo** [comment]
This has been implemented in version 1.1.4 of the `Minify` plugin.

Please note that the file you are editing must already have a valid file name with a file extension (regardless what file extension is that) so the `Minify` plugin can create a minified version of the file on the disk. So if you are editing a brand new file which hasn't been saved and named yet, this new feature shall not work.

Note: File extensions are still used to determine if the file is a valid candidate for minification or not. In fact, the file extension is check first and only if the file has a file extension something other than .css, .js, .htm, .html or .svg only then the syntax will be check to determine if the file can still be minified.
In other words, syntax checking is the secondary method. The primary method is still the file extension check.
